services/like_service.py

```
@@ -0,0 +1,70 @@
+import uuid
+from typing import Annotated
+from fastapi import Depends, HTTPException
+from datetime import datetime, timezone
+
+from repositories.like_repository import LikeRepository
+from repositories.post_repository import PostRepository
+
+
+class LikeService:
+    def __init__(self,
```

Ruff (0.14.13)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)